feat(*): update php and output

Joshua-Anderson opened the pull request:
This PR:
- pins php. It was accidentally set to`>=`, meaning that it would use the latest version of php, as opposed to the stable version.
- Updates slim from v2->v3
- Adds a healthcheck url
- Updates `/` to be the same as example-go
- Adds a `.htaccess` to that every URL maps to `web/index.php`, allowing slim to use its router

dshafik commented:
@Joshua-Anderson: any reason not to use 7.0? It is the latest stable release.

Also, if memory serves this uses nginx and php-fpm so the `.htaccess` should be unnecessary, and I'm not sure why this seems to have both nginx and apache...

Joshua-Anderson commented:
> any reason not to use 7.0?

Nope, I misread php.net and though that 7.0 was still beta a 5.6 was stable. Fixed!

> Also, if memory serves this uses nginx and php-fpm so the .htaccess should be unnecessary, and I'm not sure why this seems to have both nginx and apache...

The buildpack installs both apache and nginx in the application and lets users specify which one to use in their procfile. [heroku documentation](https://devcenter.heroku.com/articles/custom-php-settings)

Since the example app was already using apache, when I added multiple routes I added the `.htaccess` recommended by slim to make it work.

Disclaimer: I know almost nothing about PHP, I was just trying to get this up to date.

Thanks for looking over this and catching my mistakes!
